stable/artifactory/templates/artifactory-migration-scripts.yaml

```diff
@@ -2747,6 +2747,23 @@ data:
         echo "";
         done
     }
+
+    # Trim masterKey if its generated using hex 32 
+    trimMasterKey () {
+        local masterKeyDir=/opt/jfrog/artifactory/var/etc/security
+        local oldMasterKey=$(<${masterKeyDir}/master.key)
+        local oldMasterKey_Length=$(echo ${#oldMasterKey})
+        local newMasterKey=
+        if [[ ${oldMasterKey_Length} == "64" ]]; then
```

Comment:
I would test to see if it's more than 32 and not exactly 64 as users might have not exactly 64 chars.